common/djangoapps/student/tests/test_enrollment.py

```diff
@@ -286,7 +286,7 @@ def test_upgrade_proctoring_enrollment(self, mode):
         should be sent.
         """
         with patch(
-            'common.djangoapps.student.models.send_proctoring_requirements_email',
+            'common.djangoapps.student.emails.send_proctoring_requirements_email',
             return_value=None
```

Comment:
This change and the next look fishy. I'll have to revisit.

Reply:
I verified this change and it's fine. The pathname in `patch()` needs to exactly match how the symbol is used in the target code, and `CourseEnrollment` imports `send_proctoring_requirements_email` from the source (i.e., where the method is defined), `student.emails`. What's more, unit tests pass this way, and fail with a `patch()` based on `student.models`.